include/eos/fitting/contour_correspondence.hpp

```diff
@@ -78,7 +78,7 @@ struct ModelContour
 
     // We store r/l separately because we currently only fit to the contour facing the camera.
     // Also if we were to fit to the whole contour: Be careful not to just fit to the closest. The
-    // "invisible" ones behind might be closer on an e.g 90� angle. Store CNT for left/right side separately?
+    // "invisible" ones behind might be closer on an e.g 90° angle. Store CNT for left/right side separately?
```

Comment:
I'm not sure what's the change on this line - maybe it's a non-ASCII degree sign (°)? It would be great if you could remove all "zero-changes", it would make the PR a bit shorter.

Reply:
OK